docs/zh-tw/model-deployment-prepackaged-server-custom-cht.md

```diff
@@ -0,0 +1,24 @@
+---
+id: model-deployment-prepackaged-server-custom-cht
+title: Custom Pre-packaged Server
+---
+
+<div class="ee-only tooltip">Enterprise
+  <span class="tooltiptext">Applicable to Enterprise tier only</span>
+</div>
+
+當現有提供的 Pre-packaged server 滿足不了實際需求時，我們可以進階到為需求客製自己的 Pre-packaged server 由 Model URI 載入模型檔。
+
+以下為新手的建議步驟：
+
+1. 閱讀文件 [Package from Language Wrapper for python](model-deployment-language-wrapper-intro). 此文件敍述基本的 pre-packaged server。
```

Comment:
`model-deployment-language-wrapper-intro` should change to `model-deployment-language-wrapper-intro-cht`